File: steam-games-api/models/game.py
from pydantic import BaseModel

class Game(BaseModel):
    id: int # Game identifier
    name: str # Game name
    description: str # Game description
    genre: str # Game genre
    age_rating: int # Game age rating
    developer: str # Game developer

    @classmethod
    def from_dict(cls, data: dict):
        return cls(
            id=data.get("id"),
            name=data.get("name"),
            description=data.get("description"),
            genre=data.get("genre"),
            age_rating=data.get("age_rating"),
            developer=data.get("developer")
        )

File: mongo-api/models/Game.js
const mongoose = require('mongoose');

const GameSchema = new mongoose.Schema({
  id: { type: Number, required: true, unique: true },
  name: { type: String, required: true },
  description: { type: String, required: true },
  genre: { type: String, required: true },
  age_rating: { type: Number, required: true },
  developer: { type: String, required: true }
});

module.exports = mongoose.model('Game', GameSchema);